test/vega_lite/data_test.exs

```diff
@@ -0,0 +1,286 @@
+defmodule VegaLite.DataTest do
+  use ExUnit.Case
+
+  alias VegaLite.Data
+  alias VegaLite, as: Vl
+
+  @data [%{"height" => 170, "weight" => 80}, %{"height" => 190, "weight" => 85}]
```

Comment:
```suggestion
  @data [%{"height" => 170, "weight" => 80, "unused" => "a"}, %{"height" => 190, "weight" => 85, "unused" => "b"}]
```

Let's add a unused field because it will help us find places where the unused field logic is wrong. :)